Sources/Stripe/Model.swift

```diff
@@ -5,6 +5,7 @@ import TaggedMoney
 
 public struct Card: Codable, Equatable {
   public var brand: Brand
+  public var country: Country
```

Comment:
as far as I know this is a non-optional field from Stripe

Reply:
🙏